packages/jaeger-ui/src/components/Monitor/EmptyState/index.tsx

```diff
@@ -32,7 +32,7 @@ export default class MonitorATMEmptyState extends React.PureComponent {
     return (
       <Col>
         <Row justify="center">
-          <Col span={8} offset={8}>
```

Comment:
doesn't removing `offset` change the position of the column? When I see `<Col span={8} offset={8}>` I interpret it "start from column 8 and span columns 8-15"

Reply:
Previously (in antd 3x), the `<Col span={8}>` was interpreted as, "The width of column should be 8 boxes out of the 24 boxes in a grid".
That `offset` property defines that those 8 boxes must start from the 9th box out of those 24 boxes so that basically centers the col.

In antd 4x, this behavior is controlled by `<Row justify="center">`. Don't know why it was not working previously in 3x, even though it existed before too.

Here's what the page looks like now:

![image](https://github.com/jaegertracing/jaeger-ui/assets/94157520/12a538f3-aae9-47fe-855d-f1c4ab78e0a0)